src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
```

Comment:
Class name : "Test" 

I would recommend to rename the class to **express what's the tested substance** - **class** or **business logic**
Example : CreditCardValidatorTest or CardValidityTest
Additionally when there are multiple features/classes - follow tested code package structure.

Naming precisely test class, separating into packages you also improve  
 clarity of test method names - the test method name can express explicitely tested case.